_posts/en/newsletters/2023-11-08-newsletter.md

```diff
@@ -0,0 +1,230 @@
+---
+title: 'Bitcoin Optech Newsletter #276'
+permalink: /en/newsletters/2023/11/08/
+name: 2023-11-08-newsletter
+slug: 2023-11-08-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter FIXME
+
+## News
+
+- **Mailing list hosting:** administrators for the Bitcoin-Dev mailing
+  list [announced][bishop lists] that the organization hosting the list
+  plans to cease hosting any mailing lists after the end
+  of the year.  The archives of previous emails are expected to
+  continue being hosted at their current URLs for the foreseeable
+  future.  We assume that the end of email relay also affects the
+  Lightning-Dev mailing list, which is hosted by the same organization.
+
+    The administrators sought feedback from the community about options,
+    including migrating the mailing list to Google Groups.  If such a
+    migration happens, Optech will begin using that as one of our [news
+    sources][sources].
+
+    We're also aware that,
+    in the months prior to the announcement, some well-established
+    developers had begun experimenting with discussions on the
+    [DelvingBitcoin][] web forum.  Optech will begin monitoring that
+    forum for interesting or important discussions effective
+    immediately.
+
+- **HTLC aggregation with covenants:** Johan Torås Halseth
+  [posted][halseth agg] to the Lightning-Dev mailing list a suggestion
+  for using a [covenant][topic covenants] to aggregate multiple
+  [HTLCs][topic htlc] into a single output that could be spent all at
+  once if a party knew all the preimages.  If a party only knew some of
+  the preimages, they could claim just those and then the remaining
+  balance could be refunded to the other party.  Halseth notes that this
+  would be more efficient onchain and could make it more difficult to
+  perfrom certain types of [channel jamming attacks][topic channel
+  jamming attacks].
+
+## Bitcoin Core PR Review Club
+
+*In this monthly section, we summarize a recent [Bitcoin Core PR Review
+Club][] meeting, highlighting some of the important questions and
+answers.  Click on a question below to see a summary of the answer from
+the meeting.*
+
+[Fee Estimator updates from Validation Interface/CScheduler thread][review club 28368]
+is a PR by Abubakar Sadiq Ismail (ismaelsadeeq)
+that modifies the way the transaction fee estimator integrates with the
+rest of the `bitcoind` client.
+It moves fee estimator updates from occurring synchronously during mempool
+updates (transactions being added or removed) to instead occur asynchronously.
+This is a performance improvement.
```

Comment:
Well, not really. I imagine this is actually slower overall, but we're no longer blocking the really important task of processing blocks with it.

Reply:
Ah, I see the distinction, you're right, more CPU cycles will be expended almost for sure, but it's the critical path that's faster. I'll improve this wording.